src/CoreShop/Component/Order/Repository/OrderRepositoryInterface.php

```diff
@@ -39,6 +39,8 @@ public function findExpiredCarts(int $days, bool $anonymous, bool $customer): ar
 
     public function findByCustomer(CustomerInterface $customer): array;
 
+    public function findOrdersByCustomer(CustomerInterface $customer): array;
```

Comment:
Strictly speaking, this is a breaking change, right?

Reply:
Stricly speaking yes, but since this is sort of a serious bug, it's fine for us. There is no other way of solving this than to introduce a new Interface for a feature that is expected to work